app/src/main/java/nerd/tuxmobil/fahrplan/congress/alarms/AlarmReceiver.java

```diff
@@ -44,12 +44,12 @@ public void onReceive(Context context, Intent intent) {
         MyApp.LogDebug(LOG_TAG, "Received alarm = " + intent.getAction() + ".");
 
         if (ALARM_SESSION.equals(intent.getAction())) {
-            String sessionId = intent.getStringExtra(BundleKeys.ALARM_SESSION_ID);
+            String sessionId = intent.getStringExtra(BundleKeys.Companion.getALARM_SESSION_ID());
```

Comment:
Please not! `getALARM_SESSION_ID` :see_no_evil: This can be resolved by changing the `BundleKeys` file as indicated.